WEB/Src/PerformanceCollector/Perf.Tests/QuickPulse/QuickPulseServiceClientTests.cs

```diff
@@ -2115,6 +2115,7 @@ private void ProcessRequest(HttpListener listener, AutoResetEvent ev)
                                 var invariantVersion =
                                     context.Request.Headers[QuickPulseConstants.XMsQpsInvariantVersionHeaderName];
                                 var streamId = context.Request.Headers[QuickPulseConstants.XMsQpsStreamIdHeaderName];
+                                var roleName = context.Request.Headers[QuickPulseConstants.XMsQpsRoleNameHeaderName];
```

Comment:
Are you verifying the value anywhere?

Reply:
Yes, its in the _QuickPulseServiceClientSubmitsStreamIdToServiceWithPing_ method